substrate/primitives/panic-handler/src/lib.rs

```diff
@@ -30,7 +30,7 @@ use std::{
 	cell::Cell,
 	io::{self, Write},
 	marker::PhantomData,
-	panic::{self, PanicInfo},
+	panic::{self, PanicHookInfo},
```

Comment:
Unrelated change - why is this in the PR?

Reply:
I'm ok to move this to a single PR

Reply:
We will temporarily revert this, as it causes issues with the CI.